### DISCUSSION TITLE
IGPU undervolt for 7940HS

### DISCUSSION THREAD

**Discussion by luk-011** [seerge/g-helper#3159] [Ideas]
Hi, I'm new here sorry it is the wrong section for this post.
First of all, I wanna thank you all for this amazing masterpiece of software you gave us, but the reason I'm here today is also that I'd like to see an improvement for the 7940hs owners. 
I was looking for igpu undervolt option and I saw a previous post about this where was told that the CPU didn't support it, but looking in UxTu there is an option for that so I wanted to know if is now possible to add it for this CPU

Thanks for the time spent reading :)

PS: In case it's not possible to integrate in ghelper can you tell me if I can use uxtu just for igpu and ghelper for the rest without issues? I've read they are using the same libraries.

**Comment by seerge** [comment]
@luk-011 hello 7940HS [does not support undervolting](https://github.com/seerge/g-helper/discussions/736) https://github.com/seerge/g-helper/discussions/736.

Even if you see an option in UXTU - it's most likely not working. As G-Helper  uses UXTU's library under the hood for undervolting as well :)
